docs/index.yml

```
@@ -0,0 +1,265 @@
+      summary: Learn the fundamental concepts of .NET
+      links:
+        - url: https://channel9.msdn.com/Series/NET-Core-101/What-is-NET
+          itemType: video
+          text: What is .NET?
+        - url: standard/index.md
+          itemType: overview
+          text: Overview of .NET
+        - url: standard/net-standard.md
+          itemType: overview
+          text: .NET Standard
+        - url: standard/get-started.md
+          itemType: get-started
+          text: Get started
+        - url: standard/components.md
+          itemType: concept
+          text: .NET architectural components
+        - url: framework/index.md
+          itemType: overview
+          text: .NET Framework (Windows) apps
+    # Card
+    - title: Develop .NET apps
+      summary: Start developing with .NET
+      links:
+        - url: core/sdk.md
+          itemType: download
+          text: Download the .NET Core SDK
+        - url: core/tools/index.md
+          itemType: overview
+          text: .NET Core CLI overview
+        - url: core/porting/index.md
+          itemType: how-to-guide
+          text: Porting from .NET Framework to .NET Core
+        - url: desktop-wpf/overview/index.md
+          itemType: overview
+          text: Create .NET Core desktop apps for Windows
+        - url: standard/library-guidance/index.md
+          itemType: concept
+          text: Open source library guidance
+    # Card
+    - title: .NET architectural guides
+      summary: Read foundational development and architectural guidance for .NET
+      links:
+        - url: architecture/microservices/index.md
+          itemType: architecture
+          text: ".NET Microservices: Architecture for containerized .NET apps"
+        - url: architecture/modern-web-apps-azure/index.md
+          itemType: architecture
+          text: Architect modern web applications with ASP.NET Core and Azure
+        - url: architecture/containerized-lifecycle/index.md
+          itemType: architecture
+          text: "Containerized Docker application lifecycle with Microsoft platform and tools"
+        - url: architecture/modernize-with-azure-containers/index.md
+          itemType: architecture
+          text: "Modernize existing .NET applications with Azure cloud and Windows containers"
+      # footerLink (optional)
+      footerLink:
+        url: architecture/index.yml
+        text: All architecture guides
+
+# tools section (optional)
+## Languages: C#, VB, F#, 
+tools:
+  title: Programming languages # < 60 chars (optional)
+  summary: "You can write .NET apps in C#, F#, or Visual Basic."
+  items:
+    # Card
+    - title: "A modern, object-oriented, type-safe member of the C family"
+      imageSrc: https://docs.microsoft.com/media/logos/logo_Csharp.svg
+      url: csharp/index.yml
+    # Card
+    - title: "A functional programming language for .NET"
+      imageSrc: https://docs.microsoft.com/media/logos/logo_Fsharp.svg
+      url: fsharp/index.md
```

Should this be yml too? You have a few broken links in the build report

```suggestion
      url: fsharp/index.yml
```